KifuSF/CreateDonationViewController.swift

```diff
@@ -0,0 +1,246 @@
+//
+//  CreateDonationViewController.swift
+//  KifuSF
+//
+//  Created by Alexandru Turcanu on 17/12/2018.
+//  Copyright © 2018 Alexandru Turcanu. All rights reserved.
+//
+
+import UIKit
+import LocationPicker
+
+class CreateDonationViewController: UIScrollableViewController {
+    //MARK: - Variables
+    private static let descriptionPlaceholder = "Additional Info about pick up address, hour, item, etc."
+    private let imageHelper = PhotoHelper()
```

Comment:
```suggestion
    private lazy var imageHelper = PhotoHelper()
```

Reply:
Thus, keeping the initialization of `CreateDonationViewController` smaller

Reply:
Good Point!